tests/components/konnected/test_init.py

```diff
@@ -0,0 +1,452 @@
+"""Test Konnected setup process."""
+from unittest.mock import patch
```

Comment:
Please import `patch` from `asynctest` instead. It's backwards compatible and we can simplify things if we can patch coroutines too.

Reply:
This is awesome!! Didn't realize it existed.